.github/workflows/upload-binaries.yml

```diff
@@ -37,12 +37,12 @@ jobs:
         release_name: Release ${{ needs.prechecks.outputs.package_version }}
 
   upload-binary:
-    name: Upload Assets (node ${{ matrix.node-version }})
+    name: Upload Asset
     runs-on: windows-latest
     needs: release
     strategy:
       matrix:
-        node-version: ['10', '12', '14', '15', '16']
+        node-version: ["14"]
```

Comment:
We only need to build one binary and `14` is the highest LTS.

Reply:
Yes, that's the nice thing about n-api is that the ABI (Application Binary Interface) is stable , so as new versions are released, we can still rely on the binary interface remaining the same. It's the same concept as semver except at the binary level.